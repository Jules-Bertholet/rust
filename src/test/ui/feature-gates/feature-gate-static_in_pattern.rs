static ZERO: i32 = 0;

pub fn main() {
    match 0 {
        ZERO => true,
        _ => false,
    };
}
