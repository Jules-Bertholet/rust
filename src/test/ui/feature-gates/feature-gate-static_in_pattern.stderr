error[E0658]: referencing statics in patterns is experimental
  --> $DIR/feature-gate-static_in_pattern.rs:5:9
   |
LL |         ZERO => true,
   |         ^^^^
   |
   = help: add `#![feature(static_in_pattern)]` to the crate attributes to enable

error[E0658]: referencing statics in patterns is experimental
  --> $DIR/feature-gate-static_in_pattern.rs:5:9
   |
LL |         ZERO => true,
   |         ^^^^
   |
   = help: add `#![feature(static_in_pattern)]` to the crate attributes to enable

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
